docs/Tutorials/demo/budget_with_farming.md

```diff
@@ -11,33 +11,33 @@ The [budget](https://github.com/tendermint/budget) and [farming](https://github.
 ## What does budget module do?
 
-A budget module is a Cosmos SDK module that implements budget functionality. It is an independent module from other SDK modules and core functionality is to enable anyone to create a budget plan through governance param change proposal. Once it is agreed within the community, voted, and passed, it uses the budget source address to distribute amount of coins by the rate defined in the plan to the collection address. Collecting all budgets and distribution take place every epoch blocks that can be modified by a governance proposal.
+A budget module is a Cosmos SDK module that implements budget functionality. It is an independent module from other SDK modules and core functionality is to enable anyone to create a budget plan through governance param change proposal. After it is agreed within the community, voted, and passed, it uses the budget source address to distribute amount of coins by the rate that is defined in the plan to the collection address. Collecting all budgets and distribution take place every epoch blocks that can be modified by a governance proposal.
 
-One use case is for Gravity DEX farming plan. Budget module can be used to create a budget plan that defines Cosmos Hub's FeeCollector module account where transaction gas fees and part of ATOM inflation are collected as budget source address and uses custom module account (created by budget creator) as collection address. 
+One use case is the Gravity DEX farming plan. Budget module can be used to create a budget plan that defines the Cosmos Hub FeeCollector module account where transaction gas fees and part of ATOM inflation are collected as budget source address and uses a custom module account (created by budget creator) as the collection address. 
 
-## What does farming module do?
+## What does the farming module do?
 
-A farming module is a Cosmos SDK based module that implements farming functionality that provides farming rewards to staking participants called farmers. 
+The farming module is an Cosmos SDK-based module that implements farming functionality that provides farming rewards to staking participants called farmers. 
 
 One use case is to use the module to provide incentives for liquidity pool investors for their pool participation.
 
 ## Resources
 
 **Budget module**
 
-- [Github Repo](https://github.com/tendermint/budget)
-- [Spec Docs](https://github.com/tendermint/budget/blob/main/x/budget/spec/01_concepts.md)
-- This demo version [v0.1.2](https://github.com/tendermint/budget/releases)
-- Other useful resources are available in [docs](https://github.com/tendermint/budget/blob/main/docs) folder
-- Swagger Docs [v0.1.0](https://app.swaggerhub.com/apis-docs/gravity-devs/budget/0.1.0)
+- [tendermint/budget GitHub repo](https://github.com/tendermint/budget)
+- [x/budget spec docs](https://github.com/tendermint/budget/blob/main/x/budget/spec/01_concepts.md)
+- Audit-ready [demo version](https://github.com/tendermint/budget/releases)
```

Comment:
if we omit the release from the link text, then this demo will require less maintenance, please verify that all changes for all link text are valid for your intended purpose

Reply:
I initially thought we need to specify the correct release version of the project because the document is for the demo, but I understand your point and it makes me think that the demo should work with the later versions and it gets updated accordingly.